### ISSUE TITLE
Add a functional test for NetworkPolicy

### ISSUE THREAD

**Issue by fabiand** [kubevirt/kubevirt#927]
A simple test to verify that NetworkPolicy objects work with our network would be good.

- [ ] test for adding and checking NetworkPolicy
- [ ] validate that the network plugin supports NetworkPolicy objects

**Comment by fabiand** [comment]
@duyanyan are you working on this?

**Comment by duyanyan** [comment]
@fabiand Yes, I could take it if it is not a urgent task. Maybe I need more time to do it since I've ever done that before.

**Comment by duyanyan** [comment]
https://github.com/kubevirt/kubevirt/pull/1360
